numpy/polynomial/tests/test_chebyshev.py

```diff
@@ -111,6 +113,15 @@ def test_chebdiv(self):
                 res = cheb.chebadd(cheb.chebmul(quo, ci), rem)
                 assert_equal(trim(res), trim(tgt), err_msg=msg)
 
+    def test_chebpow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = np.arange(i + 1)
+                tgt = reduce(cheb.chebmul, [c]*j) if j else np.array([1])
```

Comment:
Comment from the other PR still applies - I think this would be better as:
```
tgt = reduce(cheb.chebmul, [c]*j, np.array([1]))
```
which performs:
```
tgt = np.array([1])
for _ in range(j):
    tgt = cheb.chebmul(c, tgt)
```

Reply:
Huh, I tried that earlier, I must've messed up the syntax somewhere along the line. I vastly prefer this, making the corrections now.